modules/sensor-extraction/ros-to-png/src/main.py

```diff
@@ -215,8 +253,8 @@ def main(table_name, index, batch_id, bag_path, images_path, topics, encoding, t
             batch_id=args.batchid,
             index=args.index,
             table_name=args.tablename,
-            bag_path=args.localbagpath,
-            images_path=args.localimagespath,
+            bag_path="/tmp/ros.bag",
```

Comment:
Getting these as input isn't a bad idea. Especially if the files we're working with is large and we need to reference an efs/lustre mounted path instead of the temp dir

Reply:
The container assumes the data is on S3 already and contains a get object call, so I would leave it as-is for now and extend once we figure out how we want to work with external FS's